### DISCUSSION TITLE
S400 pro 体重秤 无法读取体重数据

### DISCUSSION THREAD

**Discussion by 563816752** [XiaoMi/ha_xiaomi_home#143] [Ideas]
S400 pro 体重秤 无法读取体重数据

**Comment by SusanPhevos** [comment]
体重秤是BLE直连设备，xiaomi_home目前只能从云端获取设备属性值，不支持BLE直连读取设备属性值。

**Reply by 563816752** [reply]
s400 pro 有蓝牙mesh的功能吧，数据在云端应该有备份吧

**Reply by SusanPhevos** [reply]
S400 pro只查到体脂称，没查到体重秤。请发一下设备详情页，我看看model是什么。

**Comment by SusanPhevos** [comment]
See #362
